src/test/java/Homework17.java

```diff
@@ -0,0 +1,34 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework17 extends BaseTest{
+
+    @Test
+    public void addSongToPlaylist() throws InterruptedException {
+        String textBanner = "Added 1 song into \"first playlist.\"";
+
+        enterEmail("anna.dudnik@testpro.io");
+        enterPassword("GulyalaKorova4milk!");
+        clickLogin();
+        Thread.sleep(1000);
+        //clearMyPlaylist();
+        Thread.sleep(1000);
+        searchSong("dark");
+        Thread.sleep(1000);
+        clickViewAll();
+        Thread.sleep(1000);
+        clickSong();
+        Thread.sleep(1000);
+        clickAddTo();
+        Thread.sleep(1000);
+        clickPlaylist();
+        Thread.sleep(500);
```

Comment:
Thread.sleep is generally considered as a bad practice because it introduces unnecessary delays and make your code less efficient. Instead we can use the explicit wait.

for your reference : https://www.guru99.com/implicit-explicit-waits-selenium.html